pkgs/applications/networking/instant-messengers/twinkle/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  libxml2,
+  libsndfile,
+  file,
+  readline,
+  bison,
+  flex,
+  ucommon,
+  ccrtp,
+  qt5,
+  alsaLib,
+  speex,
+  ilbc,
+  wrapQtAppsHook,
+  fetchurl,
+  bcg729 }:
+
+stdenv.mkDerivation rec {
+  pname = "twinkle";
+  version = "1.10.2";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "LubosD";
+    rev = "v${version}";
+    sha256 = "0s0gi03xwvzp02ah4q6j33r9jx9nbayr6dxlg2ck9pwbay1nq1hx";
+  };
+
+  buildInputs = [
+    libxml2
+    file # libmagic
+    libsndfile
+    readline
+    ucommon
+    ccrtp
+    qt5.qtbase
+    qt5.qttools
+    qt5.qtquickcontrols2
+    alsaLib
+    speex
+    ilbc
+  ];
+
+  patches = [
+    (fetchurl { # https://github.com/LubosD/twinkle/pull/152 patch for bcg729 1.0.2+
+      url = "https://github.com/LubosD/twinkle/pull/152.patch";
+      sha256 = "39fc6cef3e88cfca8db44612b2d082fb618027b0f99509138d3c0d2777a494c2";
+    })
+  ];
+
+  nativeBuildInputs = [ cmake bison flex wrapQtAppsHook bcg729 ];
+
+  cmakeFlags = [ "-DWITH_G729=On" "-DWITH_SPEEX=On" "-DWITH_ILBC=On" /* "-DWITH_DIAMONDCARD=On" broken code */ ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "Twinkle is a SIP-based VoIP client";
```

Comment:
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does. 

In this case: "A SIP-based VoIP client"